spec/graphql/schema/input_object_spec.rb

```diff
@@ -237,8 +238,6 @@ def self.object_from_id(id, ctx)
         def self.resolve_type(type, obj, ctx)
           type
         end
-
-        orphan_types [Jazz::InstrumentType]
```

Comment:
TODO: could this change be addressed by adding a check for `orphan_type?` inside Subset?

Reply:
Improved `loadable?` in ae4874452